android/tangram/src/com/mapzen/tangram/geometry/Geometry.java

```diff
@@ -0,0 +1,35 @@
+package com.mapzen.tangram.geometry;
+
+import java.util.Map;
+
+/**
+ * Geometry is an abstract container of LngLat points and associated properties
+ */
+public abstract class Geometry {
+
+    protected double[] coordinates;
+    protected int[] rings;
+    protected String[] properties;
+
+    public double[] getCoordinateArray() {
```

Comment:
Ah thanks for pointing this out, I actually think the classes in `com.mapzen.tangram.geometry` don't even need to be public, I'll change that.